src/libtriton/format/pe/pe.cpp

```diff
@@ -0,0 +1,220 @@
+
+          if (exportStart == 0) {
+              std::memset(&exportTable, '\0', 40);
+              return;
+          }
+
+          exportTable.parse(raw+getOffsetFromAddress(exportStart));
+          exportTable.setName((char*)raw+getOffsetFromAddress(exportTable.getNameRVA()));
+          triton::uint32 addrTableStart = getOffsetFromAddress(exportTable.getExportAddressTableRVA());
+
+          std::vector<PeExportEntry> entries;
+          for (triton::usize i=0; i<exportTable.getAddressTableEntries(); ++i) {
+              PeExportEntry entry;
+              triton::uint32 exportRVA;
+              std::memcpy(&exportRVA, raw+addrTableStart+4*i,4);
+              if (exportRVA >= exportStart && exportRVA < exportStart+exportSize) {
+                  entry.isForward = true;
+                  entry.forwarderRVA = exportRVA;
+                  entry.forwarderName = std::string((char*)raw+getOffsetFromAddress(exportRVA));
+              } else {
+                  entry.isForward = false;
+                  entry.exportRVA = exportRVA;
+              }
+              entries.push_back(entry);
+          }
+
+          triton::uint32 nameTableStart = getOffsetFromAddress(exportTable.getNamePointerRVA());
+          triton::uint32 ordTableStart = getOffsetFromAddress(exportTable.getOrdinalTableRVA());
+          for (triton::usize i=0;i<exportTable.getNumberOfNamePointers(); ++i) {
+              triton::uint16 ordinal;
+              std::memcpy(&ordinal, raw+ordTableStart+2*i,2);
+              triton::uint32 nameRVA;
+              std::memcpy(&nameRVA, raw+nameTableStart+4*i,4);
+              entries[ordinal].ordinal = ordinal;
+              entries[ordinal].exportNameRVA = nameRVA;
+              entries[ordinal].exportName = std::string((char*)raw+getOffsetFromAddress(nameRVA));
+          }
+          for (const PeExportEntry &entry : entries)
+              exportTable.addEntry(entry);
+      }
+
+      void Pe::initImportTable(void) {
+          triton::uint32 importStart = header.getDataDirectory().getImportTable_rva();
+          if (importStart == 0)
+              return;
+          triton::uint32 importOffset = getOffsetFromAddress(importStart);
+          triton::uint32 format = header.getOptionalHeader().getMagic();
+          triton::uint64 byNameMask = (format == PE_FORMAT_PE32PLUS ?
+            0x8000000000000000 : 0x80000000);
+          triton::uint32 entrySize = (format == PE_FORMAT_PE32PLUS ? 8 : 4);
+          triton::uint32 pos = importOffset;
+          while(true) {
+            PeImportDirectory impdt;
+            if (!impdt.parse(raw+pos)) break;
+            impdt.setName((char*)raw+getOffsetFromAddress(impdt.getNameRVA()));
+            triton::uint32 impLookupTable = getOffsetFromAddress(impdt.getImportLookupTableRVA());
+            triton::uint64 importEntry  = 0;
+            std::memcpy(&importEntry, raw+impLookupTable, entrySize);
```

Comment:
The same variable (importEntry) could be either 64 or 32 bits in size depending on the format. So I zero it out and then only copy the necessary number of bytes. The best I could do is change the definition of entrySize to:

triton::uint32 entrySize = (format == PE_FORMAT_PE32PLUS ? sizeof(triton::uint64) : sizeof(triton::uint32))